prereise/cli/data_sources/wind_data.py

```diff
@@ -85,4 +85,7 @@ def extract(self, region, start_date, end_date, file_path, **kwargs):
         )
         if len(missing) > 0:
             logging.warning(f"There are {len(missing)} files missing")
+
+        # Imputing any missing data in place
+        impute.gaussian(data, wind_farms, inplace=True)
```

Comment:
@rouille is this what you had in mind (for completing the wind data download from RAP)?

Reply:
Yes. I would say it can be moved within the `if` statement with a message stating that imputation of missing data will be performed

Reply:
Perhaps, we can have in the command line a boolean `impute` argument that will trigger the imputation of missing data (True, default) or not (False) just in case the user has a homemade algorithm he/she/they would rather apply

Reply:
good point, done.